packages/relayer-lib/src/tx_builder/cosmos_to_eth.rs

```diff
@@ -280,4 +284,55 @@ where
         .calldata()
         .to_vec())
     }
+
+    #[tracing::instrument(skip_all)]
+    async fn update_client(&self, dst_client_id: String) -> Result<Vec<u8>> {
+        let now_since_unix = std::time::SystemTime::now().duration_since(std::time::UNIX_EPOCH)?;
+
+        let client_state = self.client_state(dst_client_id.clone()).await?;
+        let trusted_block_height = client_state.latestHeight.revisionHeight;
+
+        let trusted_light_block = self
+            .tm_client
+            .get_light_block(Some(trusted_block_height))
+            .await?;
+
+        let latest_light_block = self.tm_client.get_light_block(None).await?;
+
+        tracing::info!(
```

Comment:
This is not necessarily super useful without some more context, I think at the very least the client_id should be there too. Same goes for the other ones.